### PR TITLE
Feature/slippage settings

### PR DESCRIPTION
**Pull request by pacxiu**
# IMPORTANT: TO BE MERGED AFTER SLIPPAGE CALCULATIONS ARE REIMPLEMENTED

# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Review by sennett — DISMISSED** [review]

File: features/openGuniVault/openGuniVault.ts [hunk 1]
```diff
@@ -376,20 +367,39 @@ export function createOpenGuniVault$(
...
-                    const stateSubject$ = new BehaviorSubject<OpenGuniVaultState>(initialState)
+                    const stateSubject$ = new Subject<OpenGuniVaultState>()
+                    const stateSubjectShared$ = stateSubject$.pipe(shareReplay(1))
+
+                    const environmentChanges$ = merge(
+                      slippageChange$(slippageLimit$),
+                      priceInfoChange$(priceInfo$, token),
+                      balanceInfoChange$(balanceInfo$, token, account),
+                      createIlkDataChange$(ilkData$, ilk),
+                      // createInitialQuoteChange(exchangeQuote$, tokenInfo.token1),
```
Comment (johnnieskywalker) [inline features/openGuniVault/openGuniVault.ts:378]: if commented out code is no longer valid please delete it

**Comment by johnnieskywalker** [comment]
image

**Comment by johnnieskywalker** [comment]
@piekczyk fixed 
image

**Comment by johnnieskywalker** [comment]
MAX ALLOWED IS NOW 0.50% 
image

**Review by johnnieskywalker — APPROVED** [review]

**Pull request merged**